### ISSUE TITLE
Build Errors After Adding ng2-pdf-viewer to Angular 11 Project

### ISSUE THREAD

**Issue by zepuka** [VadimDez/ng2-pdf-viewer#731]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

Followed the instructions on ng2-pdf-viewer readme to install to the project using command
`npm install ng2-pdf-viewer --save`

My project's package.json now has a 'dependencies' entry of `"ng2-pdf-viewer": "^6.4.0",`

And imported the module in my app.module.ts file:
```
import { PdfViewerModule } from 'ng2-pdf-viewer';
...
@NgModule({
  ...
  imports: [
    ...
    PdfViewerModule,
  ]
})
export class AppModule { }
```

Now when I `ng build`, I get the following output:

```
Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.component.d.ts:5:10 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFDocumentProxy'.

5 import { PDFDocumentProxy, PDFSource, PDFProgressData } from 'pdfjs-dist';
           ~~~~~~~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.component.d.ts:5:28 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFSource'.

5 import { PDFDocumentProxy, PDFSource, PDFProgressData } from 'pdfjs-dist';
                             ~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.component.d.ts:5:39 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFProgressData'.

5 import { PDFDocumentProxy, PDFSource, PDFProgressData } from 'pdfjs-dist';
                                        ~~~~~~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:1:10 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFJSStatic'.

1 import { PDFJSStatic } from 'pdfjs-dist';
           ~~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:10 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFJSStatic'.

7 export { PDFJSStatic, PDFDocumentProxy, PDFViewerParams, PDFPageProxy, PDFSource, PDFProgressData, PDFPromise } from 'pdfjs-dist';
           ~~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:100 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFPromise'.

7 export { PDFJSStatic, PDFDocumentProxy, PDFViewerParams, PDFPageProxy, PDFSource, PDFProgressData, PDFPromise } from 'pdfjs-dist';
                                                                                                     ~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:23 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFDocumentProxy'.

7 export { PDFJSStatic, PDFDocumentProxy, PDFViewerParams, PDFPageProxy, PDFSource, PDFProgressData, PDFPromise } from 'pdfjs-dist';
                        ~~~~~~~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:41 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFViewerParams'.

7 export { PDFJSStatic, PDFDocumentProxy, PDFViewerParams, PDFPageProxy, PDFSource, PDFProgressData, PDFPromise } from 'pdfjs-dist';
                                          ~~~~~~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:58 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFPageProxy'.

7 export { PDFJSStatic, PDFDocumentProxy, PDFViewerParams, PDFPageProxy, PDFSource, PDFProgressData, PDFPromise } from 'pdfjs-dist';
                                                           ~~~~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:72 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFSource'.

7 export { PDFJSStatic, PDFDocumentProxy, PDFViewerParams, PDFPageProxy, PDFSource, PDFProgressData, PDFPromise } from 'pdfjs-dist';
                                                                         ~~~~~~~~~


Error: node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:83 - error TS2305: Module '"../../../../pdfjs-dist/types/pdf"' has no exported member 'PDFProgressData'.

7 export { PDFJSStatic, PDFDocumentProxy, PDFViewerParams, PDFPageProxy, PDFSource, PDFProgressData, PDFPromise } from 'pdfjs-dist';
```

Here is my `ng version` output for reference:

```
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 11.1.2
Node: 10.24.0
OS: linux x64

Angular: 11.1.2
... animations, cdk, cli, common, compiler, compiler-cli, core
... forms, material, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1101.2
@angular-devkit/build-angular   0.1101.2
@angular-devkit/core            11.1.2
@angular-devkit/schematics      11.1.2
@schematics/angular             11.1.2
@schematics/update              0.1101.2
rxjs                            6.6.3
typescript                      4.1.3
```

Update: Also tried using older version of ng2-pdf-viewer (6.3.2, 6.2.0, 6.1.2) and made sure to remove the '^' from the package.json to ensure that specific version was used, but all resulted in the same `ng build` failures.

**Comment by nmodly** [comment]
I'm also seeing this error today, although when I last built my project on Friday it didn't throw this error. Even with an identical package and package.lock in my project.

**Comment by hajaniain** [comment]
@zepuka Did you try to remove your node_modules folder?

**Comment by zepuka** [comment]
> @zepuka Did you try to remove your node_modules folder and your package-lock.json?

@hajaniain The build works after doing that, thanks! Out of curiosity, what about those files caused the build to fail? The library I brought in was completely new to the project and there wasn't a lick of pdf-related libraries in the project ever before this...

**Comment by zepuka** [comment]
> > @zepuka Did you try to remove your node_modules folder and your package-lock.json?
> 
> @hajaniain The build works after doing that, thanks! Out of curiosity, what about those files caused the build to fail? The library I brought in was completely new to the project and there wasn't a lick of pdf-related libraries in the project ever before this...

No, wait, I spoke too soon. I thought it worked 'cause I only did an npm install and an ng build after adding the dependency to the package.json. Once I import the module in my app's module, the same problem I listed originally occurs.

**Comment by hajaniain** [comment]
If you have pdfjs-dist in your package.json, remove it.

**Comment by zepuka** [comment]
> If you have pdfjs-dist in your package.json, remove it.

There we go! Yes, I had added pdfjs-dist to the package.json after the attempt to use ng2-pdf-viewer failed, so it was present when I first tried removing node_modules and package-lock.json.

So I guess the initial issue was that I needed to delete nod_modules and package-lock.json, but then the secondary issue was that I had added pdfjs-dist in the meantime.
